presto-docker-image/README.md

```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
 The default configuration uses `/data/presto` as the default for
 `node.data-dir`. Thus if using the default configuration and a mounted volume
 is desired for the data directory it should be mounted to `/data/presto`.
+
+### Quickstart
+The following steps will guide to quick test PrestoSQL using docker image.
+
+ 1. Run the PrestoSQL docker
+
+  To launch single machine prestosql instance for testing which will function as both a coordinator and worker, just run:
+
+  ```bash
+  docker run -p 8080:8080 prestosql/presto:latest
```

Comment:
- `:latest` is default, so no need to specify it here
- give the instance a name: `--name presto`